src/prpy/tsr/tsr.py

```diff
@@ -122,7 +227,7 @@ def from_json(x, *args, **kw_args):
     def to_yaml(self):
         """ Convert this TSR to a YAML string. """
         import yaml
-        return yaml.dumps(self.to_dict())
+        return yaml.dump(self.to_dict())
```

Comment:
> *yaml.dump* accepts the second optional argument, which must be an open
text or binary file. In this case,*yaml.dump* will write the produced YAML
document into the file. Otherwise, *yaml.dump* returns the produced
document.
On Jul 23, 2015 8:05 PM, "Rachel Holladay" <notifications@github.com> wrote:

> In src/prpy/tsr/tsr.py
> <https://github.com/personalrobotics/prpy/pull/159#discussion_r35385409>:
>
> > @@ -122,7 +227,7 @@ def from_json(x, *args, **kw_args):
> >      def to_yaml(self):
> >          """ Convert this TSR to a YAML string. """
> >          import yaml
> > -        return yaml.dumps(self.to_dict())
> > +        return yaml.dump(self.to_dict())
>
> @mkoval <https://github.com/mkoval> When I checked, dumps wasn't a
> command. I naively guessed @siddhss5 <https://github.com/siddhss5>
> typo-ed and meant dump.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/personalrobotics/prpy/pull/159/files#r35385409>.
>